### PR TITLE
Add missing 'acl' dependency

### PR DESCRIPTION
**Pull request by DemiMarie**
Needed for setfacl call in receiver script.

Fixes: QubesOS/qubes-issues#8426

### CODE REVIEW

**Comment by DemiMarie** [comment]
PipelineRetryFailed

**Comment by DemiMarie** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

File: rpm_spec/qubes-video-companion.spec.in [hunk 1]
```diff
@@ -36,6 +36,7 @@ Requires:       gstreamer1-plugins-good
 Requires:       python3
 Requires:       v4l-utils
 Requires:       qubes-video-companion-license
+Requires:       acl
```
Comment (marmarek) [inline rpm_spec/qubes-video-companion.spec.in:39]: It's the receiver that uses setfacl, not the sender.

**Review by marmarek — CHANGES_REQUESTED** [review]

**Review by marmarek — APPROVED** [review]

**Pull request merged**